.travis.yml

```diff
@@ -8,11 +8,11 @@ matrix:
   fast_finish: true
   include:
     - php: 7.2
-      env: ILLUMINATE_VERSION=6.0.*
```

Comment:
I think we should make these `7.*` now, or maybe even just `^7.0`

Reply:
Totally agree. Fixed. Made them `^7.0`